HACKING.md

```diff
@@ -324,61 +288,60 @@ File Attributes
   Tag_CPU_arch: v7
   Tag_FP_arch: VFPv3-D16
 ```
-### Contributing
+
+## Contributing
 
 Contributions are always welcome!
 
 Please make sure that you sign the Canonical contributor agreement [here](
 http://www.ubuntu.com/legal/contributors).
 
-Complete requirements can be found in CONTRIBUTING.md.
+Complete requirements can be found in [CONTRIBUTING.md](./CONTRIBUTING.md).
 
-Contributions are submitted through a Pull Request created from a fork of the
-`snapd` repository (under your GitHub account). 
+### Guidelines for contributors
 
-Start by creating a fork of the `snapd` repository on GitHub.
+There are few guidelines that contributors should follow:
 
-Add your fork as an additional remote to an already cloned `snapd` main
-repository. Replace `<user>` with your GitHub account username.
+- Commit messages should be well structured,  
+(_Consider complying with the [Conventional Commits](https://www.conventionalcommits.org/) specification_)
```

Comment:
@MiguelPires, we actually had an example here of how commit should like, which in lack of other definition serves as a convention. When I tried to format it a little, I was suggested here in review to point to some other sources that are updated and can be referred instead. I found something that is widely used and referred by others, which is very simplistic with no severe restrictions of any sort. It doesn't conflict with format of messages in snapd.

But the most important thing, as you might see, it is not a rule, but a thing that one may consider. It might help those who feel uncomfortable with guessing how should the message look like. If you want you can follow this conventions, if you don't then it's up to you. However, if you are confused and need some good example, this is definitely a good concise and very simple one. Wouldn't you agree?

Reply:
To be clear, I'm not saying people shouldn't use conventional commits for snapd. I just think it's strange to recommend something that (AFAIK) we don't do. Usually, we use `pkg/sub: did a thing" or just "did a thing", which people can see in snapd's git history/open PRs. Saying "Consider doing this. We don't do it but you can" seems confusing to me. But that's just my opinion, I may be wrong. If you think having this is more welcoming to newcomers to the project, then feel free to leave it :+1:

Reply:
I agree we should not recommend things we don't actually use as a team

Reply:
Guys, I understand your point. At the same time I believe we need at least some guidelines. Perhaps, it shouldn't be part of this PR and I need to bring this to Michael. I am talking from the impression that I got through my experience in these topics. If might sound weird to you, but some people that have good experience still don't know that there is a difference between description and the title inside the git message and you are supposed to place an empty line between these to. There are also some "unspoken" conventions that are not known to those who come from other VCSs (yes, there are still companies that using VCSs other than git :smiley: ).

Reply:
Sure, in this case remove it from the PR and bring the suggestion up to mvo and then we can discuss using those guidelines (or any other guidelines) specifically in a separate PR, that will unblock you from landing the rest of the changes in this PR